codex/discovery.nim

```diff
@@ -193,6 +193,18 @@ proc new*(
 
   self.updateAnnounceRecord(announceAddrs)
 
+  # --------------------------------------------------------------------------
+  # FIXME disable IP limits temporarily so we can run our workshop. Re-enable
+  #   and figure out proper solution.
+  let discoveryConfig = DiscoveryConfig(
```

Comment:
I could imagine this config object has to be passed to something, somewhere?

Reply:
Yes. And amazingly, although this patch does nothing, the CI is failing.